tests/files/README.md

```diff
@@ -0,0 +1,11 @@
+# Test files #
+
+This folder contains a collection of test files to cover different use cases of the lzma-rs library.
+
+This file describes files that are not self-explanatory in this folder.
+
+## bad-random-data
+
+This is a file that causes the code and range to be equal at some point during decoding LZMA data.
+Previously, this file would raise an `LZMAError("Corrupted range coding")`, 
+although the file is a valid LZMA file.
```

Comment:
Could you document how you generated this file? From the looks of it, it's not really "random" in the sense of `head -c N /dev/urandom`. Or is the file more structured than that (you mentioned the OpenCTM format, could you add a link to what OpenCTM is)?

My concerns are that:
- The file is quite large, which will impact the size of the repository.
- Given the file size, I also want to make sure that the license for the file's contents is compatible with the repository's license (MIT), and that it doesn't contain any sensitive/personal/private data.

Otherwise I'm wondering how many small random files would be necessary to reproduce the `code == range` case. Or whether the [dumb encoder](https://github.com/gendx/lzma-rs/blob/master/src/encode/dumbencoder.rs) could be used to generate this case with a much smaller file (e.g. with an all-zeros or all-ones file).

Reply:
Glad to hear that you are making sure it adheres to the license because I actually put down a bit of work to make it sure it does.

It is not entirely random, but not so far from it. It comes from this beauty of a Blender scene I created using a mix of the Array and Build modifier and a lot of duplication.

![image](https://user-images.githubusercontent.com/220658/70516697-bb6f3480-1b37-11ea-806f-b968d1cd8052.png)

It has subsequently been optimized and packed into multiple OpenCTM files. After that, I manually extracted the LZMA-encoded part (the vertices) of a bad file and added the unpacked size to the binary, which is missing in OpenCTM (see #17).

The reason the file was created was exactly because I wanted to reproduce the problem in a file that did not contain sensitive or private data. The problem was originally discovered in other files that I do not have the liberty to share freely ;)

So the licensing is no problem - we are happy to share the attached file under the MIT license or any other license of your choosing.

I am not sure if I am able to reproduce the issue in a smaller file. I tried writing some "random" data with a similar structure (a lot of zeros at the start of the file), but was unable to reproduce the issue.